### PR TITLE
Regenerate SSH key

### PR DESCRIPTION
**Pull request by edmunteanu**

### CODE REVIEW

**Review by odthoma — APPROVED** [review]

File: eduard.pub [hunk 1]
```diff
@@ -1 +1 @@
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC3H9TNFejSIOechVI1QgygxN+ShfbEXVvqpnkEx20YQtTE2uE8aXt4CdZ+ob082T4JzaXzTKMFoBW3u2ClkWqapjj0oBc4uA8yvs8jtCes27elhXt8z8YKsYifk+ra6ID8s6aBNZ5ztlNxdD0Pb3XqlFL9J9ouOLNJ3+oZA5fceHD3G9zCS2aR9aQhZuE+u28GkhEDthe/f73A7cv7mSnJzmK4pVo/ivRzjCI5P1TPwdW5uYv9r3Zw1dTW1zQghBVvg1LpSOyQFBk4hSTiSO1RO6hCnlsjznmPHtUHsOvChcR0XCsxm+L1jyi005GpZI5V04AsYaA7YusJQPTGLu8YF7Hwh3zLYBWdoiD05tKGSXBs1mwuWO5XOMqM2V4OrgCk6qd4UB5WldjyyNg/i1JGu8H/C7M3iMWBR2W5XsbXFZTEtqzUzD590ZF54bWnuaikqfOEEYnuyEC52hIA6wMRfHUazZV06K5+spxxPAS/R3iDC7mGTfxZiSN5QG4PCLC8uyeCUuy38fGs6Ef4u76fVWKU1iSaarqFUcb6yaCmnctMRTI8+ZZhQaC69+iIT6hdMq6tT+yZBujOJ0/s+jhV2YH8WfijxRcJPFJqU/sPDqLLcO7m7idwXplaEWOeWw4lpZRE0chGu/2w1ySzfXuaMkhnU9RSEAE3WRTzk5HhKQ== eduard.munteanu@renuo.ch
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPYjGG+4dIT2vndO8mJUIx9ED8H//fcmzCrvcqXEcuBw ed.munteanu99@gmail.com
```
[the remark this answers is not in the document]
Comment (edmunteanu) [reply eduard.pub:1]: Does it have an impact which e-mail address is there?

[the remark this answers is not in the document]
Comment (odthoma) [reply eduard.pub:1]: No this is just a "comment"

[the remark this answers is not in the document]
Comment (rnestler) [reply eduard.pub:1]: Personally I use an ssh key per service and use the comment to distinguish.

At the very least I'd advice to use an ssh-key per device and never copy private keys from one machine to the other.

Then use the "comment" (which doesn't need to be valid a valid email) to identify which machine it is for key management (so if you get a new laptop you can easily remove the old public keys from your accounts).

[the remark this answers is not in the document]
Comment (edmunteanu) [reply eduard.pub:1]: Ah i see - this is a newly generated SSH key which i added to GitHub (and ofc removed the old ones that were related to my old laptop). Seeing how I have 2 different SSH keys (one for GitHub, which has the email of my personal github account as the comment, and one for GitLab, which uses my renuo.ch email address), I think this comment is okay - also helps me to easily notice which key is used where when i use `ssh-add -l`

**Pull request merged**